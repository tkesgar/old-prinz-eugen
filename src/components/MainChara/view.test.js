import { mount } from '@vue/test-utils'
import localVue from '../../utils/test/local-vue'
import ViewComponent from './view'

describe('ViewChara', () => {
  const wrapper = mount(ViewComponent, { localVue })

  test('should mount successfully', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })

  test('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
