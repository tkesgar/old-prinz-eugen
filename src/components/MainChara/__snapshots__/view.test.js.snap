// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewChara should match snapshot 1`] = `
<div>
  <h1>
    ViewChara
  </h1>
   
  <p>
    It works!
  </p>
</div>
`;
